source/core/Math.cpp

```diff
@@ -278,6 +276,16 @@ using namespace std;
 		return _NextInstruction();													\
 		}
 
+#define DECLARE_VIREO_COMPARISON_PRIMITIVES(TYPE) \
+    DECLARE_VIREO_PRIMITIVE2( IsLT0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) <  0) ) \
+    DECLARE_VIREO_PRIMITIVE2( IsGE0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) >= 0) ) \
+    DECLARE_VIREO_COMPARISON_PRIMITIVES_BASE(TYPE)
+
+#define DECLARE_VIREO_COMPARISON_PRIMITIVES_BOOLEAN() \
+    DECLARE_VIREO_PRIMITIVE2( IsLT0Boolean, Boolean, Boolean, (_Param(1) = false) ) \
+    DECLARE_VIREO_PRIMITIVE2( IsGE0Boolean, Boolean, Boolean, (_Param(1) = true) ) \
```

Comment:
LabVIEW throws type errors when you wire a boolean to IsLT0, IsLE0, IsGT0, IsGE0. If we want to fix the warnings and refactor some code, I would rather do it such that Vireo throws a compiler error for these cases on boolean input. Thoughts?